### PR TITLE
L2Warsaw is over!

### PR DESCRIPTION
**Pull request by mateuszradomski**
Since L2Warsaw and Gitcoin is over, disable the navbar.

### CODE REVIEW

File: packages/frontend/src/build/config/common.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const common: Omit<Config, 'backend'> = {
     stages: true,
     buildAllProjectPages: false,
     upcomingRollups: true,
-    multisigReport: true,
+    multisigReport: false,
```
Comment (michalsidzej) [inline packages/frontend/src/build/config/common.ts:31]: This does not look like disabling the topbar

Comment (mateuszradomski) [reply packages/frontend/src/build/config/common.ts:31]: Check out the preview. If I don't understand something we'll talk :+]

**Review by michalsidzej — APPROVED** [review]

**Review by michalsidzej — COMMENTED** [review]
Wait, but don't we want to keep the multisigReport topbar?

**Comment by mateuszradomski** [comment]
> Wait, but don't we want to keep the multisigReport topbar?

The code is removed already :(((

**Pull request merged**